extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -265,6 +269,7 @@ export function createDropdownInputInfo(view: azdata.ModelView, info: { defaultV
 	return {
 		component: dropdown,
 		getValue: async (): Promise<InputValueType> => typeof dropdown.value === 'string' ? dropdown.value : dropdown.value?.name,
+		setValue: (value: InputValueType) => dropdown.value = value?.toString(),
```

Comment:
hmm. this is not correct, right? dropdown's value could be string or azdata.CategoryValue, I don't quite remember how to set the value when the value types are CategoryValue, there should be some examples you can refer to.

Reply:
Yeah fair - I wasn't really targetting that scenario so didn't put much thought into it. Added a conversion function for that.

I considered making this a core feature in the extensibility api but figured it's an edge enough case that it wasn't worth doing something like that.